test/functional/test_framework/governance.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018-2024 The Dash Core developers
```

Comment:
```suggestion
# Copyright (c) 2024 The Dash Core developers
```

Reply:
that's an old code moved from other file

Reply:
🤷‍♂️